<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBatchQuery</class>
 <widget class="QWizard" name="DialogBatchQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HMDB - Batch Query</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelChooseQueryType">
      <property name="text">
       <string>Please choose the type of query:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboQueryType">
      <item>
       <property name="text">
        <string>Query by mass</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelSelectSourceFile">
      <property name="text">
       <string>Select a source data file:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="frameSelectSource">
      <item>
       <widget class="QLineEdit" name="textSourcePath"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSelectSourcePath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelSelectTargetFile">
      <property name="text">
       <string>Results will be saved to:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="frameSelectTarget">
      <item>
       <widget class="QLineEdit" name="textTargetPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSelectTargetPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage3">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="labelSetQueryOptions">
      <property name="text">
       <string>Customize the query options:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="frameQueryOptions">
      <widget class="QWidget" name="pageQueryMassOptions">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioAvgMass">
            <property name="text">
             <string>Averaged &amp;mass</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioMonoMass">
            <property name="text">
             <string>Monoisotopi&amp;c mass</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelMassTolerance">
            <property name="text">
             <string>Mass tolerance:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Relative (ppm)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelSetMassModification">
          <property name="text">
           <string>Modification:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="viewMassModification">
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage4">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="labelSelectQueryFields">
      <property name="text">
       <string>Select expected fields in the result:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="viewQueryFields">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage5">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="labelWaiting">
      <property name="text">
       <string>Please wait, your request is being processed...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage6">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="labelFinished">
      <property name="text">
       <string>Done.</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonViewResult">
      <property name="text">
       <string>View results</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
